import {Object.exports._errnoException} from 'util';